mayastor/src/bdev/device.rs

```diff
@@ -50,12 +50,8 @@ use crate::core::{
 static BDEV_LISTENERS: Lazy<RwLock<HashMap<String, Vec<DeviceEventListener>>>> =
     Lazy::new(|| RwLock::new(HashMap::new()));
 
-// Size of the memory pool for NVMe I/O structures.
-const IOCTX_POOL_SIZE: u64 = 64 * 1024 - 1;
-static IOCTX_POOL: Lazy<MemoryPool<IoCtx>> = Lazy::new(|| {
-    MemoryPool::<IoCtx>::create("bdev_io_ctx", IOCTX_POOL_SIZE)
-        .expect("Failed to create memory pool for bdev I/O context")
-});
+// Memory pool for bdev I/O context.
+static BDEV_IOCTX_POOL: OnceCell<MemoryPool<IoCtx>> = OnceCell::new();
```

Comment:
The drop for the mempool is never called as it's static. IMHO, the accounting it does is very useful so let us try to call it explicitly on shutdown such that we know, at least during the test that all IO's are accounted for. (we can do this in a follow up change)